examples/dask_ml_simple.py

```diff
@@ -0,0 +1,76 @@
+"""
+Optuna example that optimizes a classifier configuration for the Iris dataset using Dask-ML.
+
+In this example, we optimize a logistic regression classifier configuration for the Iris dataset.
+The classifier is from dask-ml while the dataset is from sklearn.
+We optimize the choice of solver (admm, gradient descent, or proximal_grad),
+the regularization (penalty) when relevant and its strength (C).
+
+We have the following two ways to execute this example:
+
+(1) Execute this code directly.
+    $ python dask_ml_simple.py
+
+
+(2) Execute through CLI.
+    $ STUDY_NAME=`optuna create-study --direction maximize --storage sqlite:///example.db`
+    $ optuna study optimize dask_ml_simple.py objective --n-trials=100 --study $STUDY_NAME \
+      --storage sqlite:///example.db
+
+"""
+
+import dask.array as da
+from dask_ml.linear_model import LogisticRegression
+from dask_ml.model_selection import train_test_split
+from distributed import Client
+from distributed import LocalCluster
+from sklearn.datasets import load_iris
+
+
+# FYI: Objective functions can take additional arguments
+# (https://optuna.readthedocs.io/en/stable/faq.html#objective-func-additional-args).
+def objective(trial):
+    iris = load_iris()
+    X, y = iris.data, iris.target
+    X, y = da.from_array(X, chunks=len(X) // 5), da.from_array(y, chunks=len(y) // 5)
+
+    solver = trial.suggest_categorical('solver', ['admm', 'gradient_descent', 'proximal_grad'])
+    C = trial.suggest_uniform('C', 0.0, 1.0)
+
+    if solver == 'admm' or solver == 'proximal_grad':
+        penalty = trial.suggest_categorical('penalty', ['l1', 'l2', 'elastic_net'])
+    else:
+        # 'penalty' parameter isn't relevant for this solver,
+        # so we always specify 'l2' as the dummy value.
+        penalty = 'l2'
+
+    max_iter = 100
+
+    classifier = LogisticRegression(max_iter=max_iter, solver=solver, C=C, penalty=penalty)
```

Comment:
I think the variable `max_iter` is only used here.
Please input the literal value like `LogisticRegression(max_iter=100, ...`.

Reply:
I went ahead and made it a parameter to optimize

Reply:
Thank you for your modification. But, I think that tuning of `max_iter` can be eliminated.
IMO, LogisticRegression stops the training based on the convergence criteria ( `tol` ) and sufficient large `max_iter` usually archives good results with less time compared to tuning iterations parameter.

Reply:
Ok, I'll change that

Reply:
Thanks!